compiler/test/grainfmt/records.expected.gr

```diff
@@ -76,19 +77,22 @@ record Commented { // brace comment
   // trailing
 }
 
-let x: Commented = { // comment 1
+let x: Commented = {
+  // comment 1
   longlonglongnamenamename1: "A",
   longlonglongnamenamename2: "B",
   longlonglongnamenamename3: 42,
 }
 
-let { // a comment 2
+let {
+  // a comment 2
```

Comment:
This was fixed for block but not destructuring?

Reply:
I only fixed it for the cases where we have forced hardlines—blocks. I can update how these are handled in all of the cases.